setup.py

```diff
@@ -10,7 +10,7 @@
         'osconf',
         'expects',
         'click',
-        'mamba',
+        'mamba==0.8.6',
```

Comment:
It's better to use 'mamba<0.9.0' because this includes fix releases of 0.8 series. In this case I don't think there will be any fix for 0.8 series, but for other cases...

Reply:
Okey, i'll keep it in mind!